pennylane/ops/qubit/other.py

```diff
@@ -0,0 +1,117 @@
+# Copyright 2018-2021 Xanadu Quantum Technologies Inc.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+This module contains the available built-in discrete-variable
+quantum operations supported by PennyLane, as well as their conventions.
+"""
+import functools
+
+# pylint:disable=abstract-method,arguments-differ,protected-access
+import numpy as np
+
+import pennylane as qml
+from pennylane.operation import AnyWires, Operation
+
+
+class QFT(Operation):
```

Comment:
Can anyone think of where this might fit better?

Reply:
`QFT` actually feels more like a subroutine template rather than a regular qubit operation; maybe it could be moved there? (Otherwise, I think `arithmetic` is the closest match, but even that's not quite right)

Reply:
Any reasoning as why it wasn't a subroutine template?

Reply:
Good question - calling in @trbromley for this thoughts since he wrote the `QFT` :slightly_smiling_face: